infra/scripts/cometbft.toml

```diff
@@ -34,7 +34,15 @@ dependencies = ["cometbft-pull", "docker-network-create"]
 [tasks.cometbft-node-id]
 script = """
 id=`docker exec ${CMT_CONTAINER_NAME} cometbft show-node-id`
-ip=`curl ifconfig.me`
+ip=`curl https://ifconfig.me`
+
+set +e
+echo ${ip} | grep --quiet ":"
+if [ $? = 0 ]; then
+  ip="[$ip]"
+fi
+set -e
```

Comment:
What does this do?

Reply:
It checks whether the external IP address is an ipv6 address, and if so, it adds square brackets. I will add a comment. In IPv6, you need to wrap an address to make an IP:PORT pair, as far as I am aware.

Reply:
I see, thanks. Yeah, definitely worth a comment :pray: